### ISSUE TITLE
latest cpu version tensorflow report error when run slim model

### ISSUE THREAD

**Issue by guizili0** [tensorflow/tensorflow#23091]
**System information**
- use tensorlow master branch commit 0918aa74153664dbc61604af3cf0d66eb334c7ce build the code, build command is : bazel build -c opt --copt=-L/home/guizili/tool/gcc6.3/lib64 //tensorflow/tools/pip_package:build_pip_package
- OS Platform and Distribution (CentOS Linux release 7.4.1708 (Core)):
- Python version 2.7.5:
- Bazel version (0.15.0):
- GCC/Compiler version (6.3.0):
- CUDA/cuDNN version: NA
- GPU model and memory: NA

run train command in tensorlfow model: https://github.com/tensorflow/models
command is:
train_dir="./train_dir"
    rm -rf $train_dir
    mkdir -p $train_dir
    python research/slim/train_image_classifier.py \
    --dataset_name=imagenet \
    --train_dir=$train_dir \
    --dataset_dir=/lustre/dataset/tensorflow/imagenet \
    --dataset_split_name=train \
    --model_name=inception_v3 \
    --batch_size=64 \
    --max_number_of_steps=220 \
    --clone_on_cpu=True

error log:
2018-10-19 09:29:57.155023: E tensorflow/core/framework/node_def_util.cc:110] Error in the node: {{node InceptionV3/InceptionV3/Mixed_7c/Branch_1/Conv2d_0b_1x3/Conv2D}} = Conv2D[T=DT_FLOAT, data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true](InceptionV3/InceptionV3/Mixed_7c/Branch_1/Conv2d_0a_1x1/Relu, InceptionV3/Mixed_7c/Branch_1/Conv2d_0b_1x3/weights/read)
2018-10-19 09:29:57.179106: E tensorflow/core/framework/node_def_util.cc:110] Error in the node: {{node InceptionV3/InceptionV3/Mixed_7c/Branch_1/Conv2d_0c_3x1/Conv2D}} = Conv2D[T=DT_FLOAT, data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true](InceptionV3/InceptionV3/Mixed_7c/Branch_1/Conv2d_0a_1x1/Relu, InceptionV3/Mixed_7c/Branch_1/Conv2d_0c_3x1/weights/read)
2018-10-19 09:29:57.204720: E tensorflow/core/framework/node_def_util.cc:110] Error in the node: {{node InceptionV3/InceptionV3/Mixed_7c/Branch_2/Conv2d_0a_1x1/Conv2D}} = Conv2D[T=DT_FLOAT, data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true](InceptionV3/InceptionV3/Mixed_7b/concat, InceptionV3/Mixed_7c/Branch_2/Conv2d_0a_1x1/weights/read)
2018-10-19 09:29:57.228912: E tensorflow/core/framework/node_def_util.cc:110] Error in the node: {{node InceptionV3/InceptionV3/Mixed_7c/Branch_2/Conv2d_0b_3x3/Conv2D}} = Conv2D[T=DT_FLOAT, data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true](InceptionV3/InceptionV3/Mixed_7c/Branch_2/Conv2d_0a_1x1/Relu, InceptionV3/Mixed_7c/Branch_2/Conv2d_0b_3x3/weights/read)

attach the log while run mobilenet.
[log.txt](https://github.com/tensorflow/tensorflow/files/2494158/log.txt)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
TensorFlow installed from
TensorFlow version
Exact command to reproduce
Mobile device

**Comment by guizili0** [comment]
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): NA
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: NA
- TensorFlow installed from (source or binary): source
- TensorFlow version (use command below):commit 0918aa74153664dbc61604af3cf0d66eb334c7ce
-Exact command to reproduce:
train_dir="./train_dir"
rm -rf $train_dir
mkdir -p $train_dir
python research/slim/train_image_classifier.py 
--dataset_name=imagenet 
--train_dir=$train_dir 
--dataset_dir=/lustre/dataset/tensorflow/imagenet 
--dataset_split_name=train 
--model_name=inception_v3 
--batch_size=64 
--max_number_of_steps=220 
--clone_on_cpu=True

**Comment by JanSchepers** [comment]
Any updates on this? I'm having a similar problem...

**Comment by tensorflowbutler** [comment]
Nagging Assignee @wt-huang: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by sachinprasadhs** [comment]
There has been quite a few changes made since this issue is opened, you can retry with the latest version here https://github.com/tensorflow/models/tree/master/research/slim#Training and close this issue if your issue is resolved. Thanks!

**Comment by guizili0** [comment]
> There has been quite a few changes made since this issue is opened, you can retry with the latest version here https://github.com/tensorflow/models/tree/master/research/slim#Training and close this issue if your issue is resolved. Thanks!

Thanks and close it as it too old.
